src/main/java/com/google/jenkins/plugins/k8sengine/KubectlWrapper.java

```diff
@@ -93,6 +109,26 @@ public static void runKubectlCommand(
         context.getWorkspace().child(tempFile).delete();
       }
     }
+    return output;
+  }
+
+  /**
+   * Runs the specified kubectl command within the specified {@link JenkinsRunContext}'s
+   * environment.
+   *
+   * @param context The {@link JenkinsRunContext} whose environment the command will execute in.
+   * @param kubeConfig The {@link KubeConfig} containing the credentials for the cluster being
+   *     executed against.
+   * @param command The kubectl command to be run.
+   * @param args Arguments for the command.
+   * @throws IOException If an error occurred while executing the command.
+   * @throws InterruptedException If an error occured while executing the command.
+   */
+  public static String runKubectlCommand(
```

Comment:
No need to maintain the static method, instead having a single consistent usage is preferred. Suggest removing static method and re-factoring existing calls sites to use new instance method.